### DISCUSSION TITLE
How to hide match (undeline decoration of searched text)

### DISCUSSION THREAD

**Discussion by akahanek** [davatorium/rofi#1994] [General]
Hi,
is there any option to hide the undeline decoration of searched text in the 1.7.1 version. There were -no-show-match or -show-match options in the older versions but now I cannot find similar option in the version 1.7.1.

Thank you for any help.
A.

**Comment by DaveDavenport** [comment]
yes, see https://github.com/davatorium/rofi/discussions/1347

**Reply by akahanek** [reply]
Awesome! Thank you.
